doc/api/next_api_changes/behavior/XXXXX-AL.rst

```diff
@@ -0,0 +1,11 @@
+MEP22 Tool image path semantics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Previously, MEP22 Tools would try to load their icon (``tool.image``) relative
```

Comment:
I assume almost nobody will know what *MEP22 Tools* are. Do we have a description or a link?

Reply:
Reworded.